README.md

```diff
@@ -141,19 +141,19 @@ Get REST Channel Presence:
 
 ```dart
 void getPresence([ably.RestPresenceParams params]) async {
-    // getting channel history, by passing or omitting the optional params
-    var result = await channel.presence.get(params);
-
-    var presenceMembers = result.items; //returns PresenceMessages
-    var hasNextPage = result.hasNext(); //tells whether there are more results
-    if(hasNextPage){
-      result = await result.next();  //fetches next page results
-      presenceMembers = result.items;
-    }
-    if(!hasNextPage){
-      result = await result.first();  //fetches first page results
-      presenceMembers = result.items;
-    }
+  // getting channel presence members, by passing or omitting the optional params
+  var result = await channel.presence.get(params);
+
+  var presenceMembers = result.items; //returns PresenceMessages
```

Comment:
It would be good to conform these comments to include a space after `//`, as you do elsewhere. e.g.:

```suggestion
  var presenceMembers = result.items; // returns PresenceMessages
```

Reply:
Addressed: 593cecb3cafb89b55afd70572f9dcf02616b14ab